content/21.nutraceuticals.md

```diff
@@ -177,77 +183,77 @@ Vitamin D can modulate both the adaptive and innate immune system and is associa
 This led researchers to question whether there was a seasonal component to the SARS-CoV-2 pandemic and whether vitamin D levels might play a role [@doi:10.3390/nu12040988; @doi:10.1111/joim.13149; @doi:10.1016/j.ajic.2020.06.193; @doi:10.1111/apt.15777].
 Though it is assumed that COVID-19 is seasonal, multiple other factors that can affect vitamin D levels should also be considered.
 These factors include an individual’s nutritional status, their age, their occupation, skin pigmentation, potential comorbidities, and the variation of exposure to sunlight due to latitude amongst others.
-Indeed, it has been estimated that each degree of latitude north of 28 degrees corresponded to a 4.4% increase of COVID-19 mortality, indirectly linking a persons vitamin D levels via exposure to UVB light to COVID-19 mortality [@doi:10.1111/joim.13149]. 
+Indeed, it has been estimated that each degree of latitude north of 28 degrees corresponded to a 4.4% increase of COVID-19 mortality, indirectly linking a persons vitamin D levels via exposure to UVB light to COVID-19 mortality [@doi:10.1111/joim.13149].
 
-As the pandemic evolves, further research of varying quality has investigated some of the potential links identified early in the pandemic [@doi:10.3390/nu12040988] between vitamin D and COVID-19. 
+As the pandemic has evolved, additional research of varying quality has investigated some of the potential links identified early in the pandemic [@doi:10.3390/nu12040988] between vitamin D and COVID-19. 
 Indeed, studies are beginning to investigate whether there is any prophylactic and/or therapeutic relationship between vitamin D and COVID-19.
-A study in Switzerland demonstrated that 27 SARS-CoV-2 positive patients exhibited 25-hydroxyvitamin D plasma concentrations that were significantly lower (11.1 ng/ml) than those of SARS-CoV-2 negative patients (24.6 ng/ml; _p_ = 0.004), an association that held when stratifying for patients greater than 70 years old [@doi:10.3390/nu12051359].
+A study in Switzerland demonstrated that 27 SARS-CoV-2 positive patients exhibited 25-hydroxyvitamin D plasma concentrations that were significantly lower (11.1 ng/ml) than those of SARS-CoV-2 negative patients (24.6 ng/ml; _p_ = 0.004), an association that held when stratifying patients greater than 70 years old [@doi:10.3390/nu12051359].
 These findings seem to be supported by a Belgian observational study of 186 SARS-CoV-2 positive patients exhibiting symptoms of pneumonia, where 25-hydroxyvitamin D plasma concentrations were measured and a CT scan of the lungs was obtained upon hospitalization [@doi:10.1101/2020.05.01.20079376].
 A significant difference in 25-hydroxyvitamin D levels was observed between the SARS-CoV-2 patients and 2,717 season-matched diseased controls.
+<!-- I am a little confused about the next two sentences -- I think they probably did a different comparison but I can't quite work out what it was!-->
```

Comment:
Just a note I added for you -- working on the command line is weird!

Reply:
I think they did, I think looking at the total number of patients there was a significant difference in the 25-hydroxyvitamin D, whereas looking at male and female separately, males were deficient and females were more alike the controls. Does that make sense? :/

Reply:
Yes! Thank you so much! I might propose a little addition to try to make this a little more obvious!

Reply:
Perfect :)